### PR TITLE
docs: add 10.2.8 release notes

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: docs/src/main/paradox/release-notes/10.2.x.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+via `ServerBinding.addToCoordinatedShutdown`, the server will allow ongoing requests to be completed for a configurable
+period of time while immediately rejecting incoming requests before shutting down the application process.
+This helps to avoid downtime during rolling upgrades.
+
+#### akka-http-core
+
+* Enable idle-timeout when http2 is enabled [#3965](https://github.com/akka/akka-http/pull/3965)
+* Fix h2c via Upgrade [#4000](https://github.com/akka/akka-http/pull/4000)
+* Fail pool slot if connection is canceled before it is failed [#3981](https://github.com/akka/akka-http/pull/3981)
+* Ensure PoolMaster.shutdown(…) does not hang [#4004](https://github.com/akka/akka-http/pull/4004)
```
Comment (raboof) [inline docs/src/main/paradox/release-notes/10.2.x.md:17]: ellipses :heart:

File: docs/src/main/paradox/release-notes/10.2.x.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+* Implement graceful termination for HTTP/2 servers [#4018](https://github.com/akka/akka-http/pull/4018)
+* Remove unused var from ServerTerminator [#4011](https://github.com/akka/akka-http/pull/4011)
+
+#### akka-http
+
+* Interpret SSE heartbeats as events [#4020](https://github.com/akka/akka-http/pull/4020)
+
+#### docs
+
+* Simplify bullet points in 10.2.7 release notes[#3926](https://github.com/akka/akka-http/pull/3926)
```
Comment (raboof) [inline docs/src/main/paradox/release-notes/10.2.x.md:27]:
```suggestion
* Simplify bullet points in 10.2.7 release notes [#3926](https://github.com/akka/akka-http/pull/3926)
```

**Review by raboof — APPROVED** [review]

**Review by johanandren — APPROVED** [review]

File: docs/src/main/paradox/release-notes/10.2.x.md (hunk 2 above)
Comment (jrudolph) [reply docs/src/main/paradox/release-notes/10.2.x.md:27]: Oops, too fast with the trigger. https://github.com/akka/akka-http/pull/4035

File: docs/src/main/paradox/release-notes/10.2.x.md (hunk 1 above)
Comment (jrudolph) [reply docs/src/main/paradox/release-notes/10.2.x.md:17]: As seen in the original commit text :)

**Pull request merged**